src/test/java/org/apache/commons/beanutils2/converters/ZoneIdConverterTestCase.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.commons.beanutils2.converters;
+
+
+import java.time.ZoneId;
+
+import org.apache.commons.beanutils2.ConversionException;
+import org.apache.commons.beanutils2.Converter;
+
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+
+/**
+ * Test Case for the ZoneIdConverter class.
+ *
+ */
+public class ZoneIdConverterTestCase extends TestCase {
+
+    public static TestSuite suite() {
+        return new TestSuite(ZoneIdConverterTestCase.class);
+    }
+
+    // ------------------------------------------------------------------------
```

Comment:
I would remove all such // comments. These feel useless and noisy to me in the age of IDEs.

Reply:
Do you mind if I do the removal of these comments in another PR?  I plan on doing it to the whole code base and not just these classes.

Reply:
Sounds good to me :-)